astropy/_erfa/core.py.templ

```diff
@@ -38,15 +38,12 @@ from astropy.utils.exceptions import ErfaError, ErfaWarning
 from astropy.utils.misc import check_broadcast
 
 from . import ufunc
-from .ufunc import (dt_eraASTROM, dt_eraLDBODY, dt_pv,
```

Comment:
Removed imports here since they are not used anyway, and make much more sense in the `_erfa` module.